### DISCUSSION TITLE
Running server log and server config file?

### DISCUSSION THREAD

**Discussion by maldonadojuan** [Nimdy/Dedicated_Valheim_Server_Script#87] [Q&A]
Is there a file that you can tail / view that shows live actions in the server like users connecting?

Also is there a location where server memory is located? I have allocated over 6 gigs to my virtual machine, but the info command within the game says it is using 165 mb allocated memory. TOP and free -m show somewhere in the neighborhood of 1800mb ram usage. Is there a way to get an accurate count and set a dedicated value for the server to utilize like back in the Minecraft server days?

Thanks in advance!

**Comment by Nimdy** [comment]
sudo cat /var/log/syslog | grep ZDOID

for the memory.. not sure... its so sus... I dont even know

**Reply by rautamiekka** [reply]
`sudo grep ZDOID /var/log/syslog`, no need for useless `cat` piping.

**Reply by Nimdy** [reply]
HOW DARE YOU! :P

**Reply by maldonadojuan** [reply]
Thank you all very much! Any info on the memory Nimby? :) thank you for building this by the way. It is extremely slick!

**Reply by Nimdy** [reply]
nothing yet... its hit or miss....
